src/lib/api/KlefkiAPI.ts

```diff
@@ -34,11 +39,34 @@ export const getNotionEventPage = async (pageUrl: URL): Promise<NotionEventDetai
   return response.data;
 };
 
-export const createDiscordEvent = async (event: CreateDiscordEventRequest): Promise<void> => {
+export const createDiscordEvent = async (event: CreateDiscordEventRequest): Promise<any> => {
```

Comment:
`Promise<any>` scares me, I think we have a type file for types for Klefki responses, let's add a type there for the desired response type

Reply:
Same goes for the other API responses